sw_webpush.js

```diff
@@ -37,13 +37,19 @@ self.addEventListener('activate', function (event) {
 })
 
 self.addEventListener('push', function (event) {
+  var now = new Date()
+  var currentHour = now.getHours()
+  var currentMinute = now.getMinutes()
+  var currentSecond = now.getSeconds()
+  const formattedTime = currentHour + '_' + currentMinute + '_' + currentSecond
   console.log('Push event: ', event)
   // get all the notification data
   var notificationData = JSON.parse(event.data.text())
   var title = notificationData.title
   var notificationOptions = notificationData.notificationOptions
   var data = notificationOptions.data
   var key
+  data.wzrk_id = data.wzrk_id + formattedTime
```

Comment:
can we do  data.wzrk_id += new Date().getTime() ??

Also what;s the type of data.wzrk_id ?

Reply:
you can prefix the timestamp with _ if you wish to

Reply:
Sure, I'm trying this "data.wzrk_id += new Date().getTime() "
Also the type of "data.wzrk_id " is string